smac/intensification/successive_halving.py

```diff
@@ -26,6 +26,25 @@ class SuccessiveHalving(Intensifier):
     Implementation from "BOHB: Robust and Efficient Hyperparameter Optimization at Scale" (Falkner et al. 2018)
     (refer supplementary)
```

Comment:
Can we add this link http://proceedings.mlr.press/v80/falkner18a/falkner18a-supp.pdf here? That'll make it easier to find what we're referring to. Also, could you please write "Implementation following the description in ..."?